src/main/webapp/app/modules/canvas/filter/filter-panel.tsx

```diff
@@ -75,19 +93,84 @@ const FilterPanel = (props: IFilterPanelProp) => {
       </div>
     </>
   );
-};
 
+
+ return (
+   <>
+     <div className={!isFilterPanelClose ? 'FilterPanel-Main FilterPanel-hide' : 'FilterPanel-Main FilterPanel-show'}>
+       <div className={isFilterMinimize ? 'FilterPanel FilterPanel-minimize' : 'FilterPanel FilterPanel-maximize'}>
+         <PanelHeader setMinimize={setFilterMinimize} isMinimized={isFilterMinimize} titleKey="entity.action.filter" />
+         <Divider size={'S'} />
+         <Flex direction="column" gap="size-100" justifySelf="center">
+           <View>
+             <div className="filter-body">
+               {props.featuresList &&
+                 props.featuresList.length > 0 &&
+                 props.featuresList.map((item, i) => {
+                   if (item.featureType === 'DIMENSION') {
+                     return <FilterElement key={item.id} feature={item} />;
+                   }
+                 })}
+             </div>
+             <Flex direction="row" justifyContent="end" marginTop="size-125" marginBottom="size-125">
+               <Button
+                 onPress={() => {
+                   if (!visualizationId) {
+                     props.applyFilter(props.selectedFilters, props.visualmetadata, props.view);
+                   } else {
+                     props.applyFilterForShareLink(props.selectedFilters, props.visualmetadataEntity, props.view);
```

Comment:
why all lines got changed...what is the change here

Reply:
done